terraform/standalone/nms/gcp/terraform.tfvars.example

```diff
@@ -1,4 +1,5 @@
 image_name           = "myimagename"
+network_name         = "vpc_network_name"
```

Comment:
should this be `default` rather than `vpc_network_name` to match the previous behaviour?